src/block/horizontal-scroller/edit.js

```diff
@@ -0,0 +1,190 @@
+import classnames from 'classnames'
+import {
+	 ColumnInnerBlocks,
+	 GroupPlaceholder,
+	 InspectorTabs,
+	 AdvancedRangeControl,
+	 InspectorStyleControls,
+	 PanelAdvancedSettings,
+	 AdvancedSelectControl,
+} from '~stackable/components'
+
+import {
+	 BlockDiv,
+	 useGeneratedCss,
+	 MarginBottom,
+	 Alignment,
+	 getAlignmentClasses,
+	 Advanced,
+	 CustomCSS,
+	 Responsive,
+	 CustomAttributes,
+	 EffectsAnimations,
+	 ConditionalDisplay,
+	 Transform,
+	 ColumnsControl,
+	 getContentAlignmentClasses,
+} from '~stackable/block-components'
+import { useBlockContext } from '~stackable/hooks'
+import {
+	 withBlockAttributeContext,
+	 withBlockWrapperIsHovered,
+	 withQueryLoopContext,
+} from '~stackable/higher-order'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+
+const Edit = props => {
+	 const {
+		 className,
+		 clientId,
+		 isSelected,
+	 } = props
+
+	 useGeneratedCss( props.attributes )
+
+	 const blockAlignmentClass = getAlignmentClasses( props.attributes )
+	 const { hasInnerBlocks } = useBlockContext()
+	 const [ columnProviderValue, columnTooltipClass ] = ColumnInnerBlocks.useContext()
+
+	 const blockClassNames = classnames( [
+		 className,
+		 'stk-block-horizontal-scroller editor',
+		 columnTooltipClass,
+	 ] )
+
+	 const contentClassNames = classnames( [
+		'stk-inner-blocks',
+		blockAlignmentClass,
+		'stk-block-content',
+	 ], getContentAlignmentClasses( props.attributes, 'horizontal-scroller' ) )
+
+	 return (
+		 <>
+			 { isSelected && (
+				 <>
+					 <InspectorTabs />
+
+					 <Alignment.InspectorControls hasRowAlignment={ true } />
+					 <BlockDiv.InspectorControls />
+					 <Advanced.InspectorControls />
+					 <Transform.InspectorControls />
+					 <EffectsAnimations.InspectorControls />
+					 <CustomAttributes.InspectorControls />
+					 <CustomCSS.InspectorControls mainBlockClass="stk-block-horizontal-scroller" />
+					 <Responsive.InspectorControls />
+					 <ConditionalDisplay.InspectorControls />
+					 <InspectorStyleControls>
+						<PanelAdvancedSettings
+							title={ __( 'Horizontal Scroll', i18n ) }
+							id="horizontal-scroll"
+							initialOpen={ true }
+						>
+							<ColumnsControl />
+							<AdvancedSelectControl
+								label={ __( 'Snapping', i18n ) }
+								attribute="horizontalScrollerSnap"
+								options={ [
+									{ value: 'start', label: __( 'Snap to Start', i18n ) },
+									{ value: 'center', label: __( 'Snap to Center', i18n ) },
+									{ value: 'none', label: __( 'No Snapping', i18n ) },
+								] }
+								value={ props.attributes.horizontalScrollerSnap || 'center' }
+								default="center"
+							/>
+							<AdvancedRangeControl
+								label={ __( 'Item Width', i18n ) }
+								attribute="horizontalScrollerColumnWidth"
+								responsive="all"
+								units={ [ 'px', 'em', '%' ] }
+								min={ [ 0, 0, 0 ] }
+								sliderMax={ [ 100, 1000, 1000 ] }
```

Comment:
These values seem too small or large.

What we want to achieve with our `AdvancedRangeControl` is that when you drag the handle to min and drag the handle to max, they shouldn't look too extreme / broken.

Try using max values of `[ 500, 40, 50 ]`